BUILD

```diff
@@ -29,19 +27,15 @@ cc_library(
     ],
 )
 
-load("@rules_python//python:packaging.bzl", "py_wheel")
-
-load(":package.bzl", "py_package")
```

Comment:
By reading the code, I had the feeling that the `package.bzl` from this repo had no effect other than calling the function of the same name in `rules_python` (plus *allowing* but not *using* the `prefix` parameter), so I removed it.